runtime.go

```diff
@@ -691,6 +695,20 @@ func (c ComponentScript) Render(ctx context.Context, w io.Writer) error {
 
 // RenderScriptItems renders a <script> element, if the script has not already been rendered.
 func RenderScriptItems(ctx context.Context, w io.Writer, scripts ...ComponentScript) (err error) {
+	return RenderScriptItemsWithSpread(ctx, w, nil, scripts...)
```

Comment:
I mainly only went this route to keep `RenderScriptItems` unchanged to avoid a breaking change. But if that's not a concern we can just add the extra parameter to this function.